editor/modes/visual-editor/index.js

```diff
@@ -57,6 +57,7 @@ class VisualEditor extends Component {
 		if (
 			! isEditableElement( document.activeElement ) &&
 			( event.ctrlKey || event.metaKey ) &&
+			! event.shiftKey &&
```

Comment:
I would prefer that we add a utility function similar to the one in TinyMCE core, because I think we'll need to do this in a few other (future) places. It's also good to describe with the function name what exactly we are blocking. In some cases this would be only the meta key, in others all modifiers. https://github.com/tinymce/tinymce/blob/4.6.4/src/core/src/main/js/util/VK.js